### ISSUE TITLE
Surge Bridge

### ISSUE THREAD

**Issue by A5TR0spud** [Anuken/Mindustry-Suggestions#3518]
### Describe the content or mechanics you are proposing.

Based on a suggestion in the discord
Personally I'm indifferent to it, but as I write this I get more biased
Note: After writing this the idea in the discord has a few variations and general changes. I will not be addressing them.

An item bridge that lets you place the exit point from any angle, similar to a power node. Same range as normal bridge, maybe less. Items are outputted either as if the exit is a router or only 2 sides max.

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

Small inconvenient corners that would normally require a huge rerouting to avoid something (like a factory, especially in multiplayer) or 2 mass drivers just a few blocks a way, can instead have a surge bridge to connect the diagonally adjacent tiles without worrying about the 2 blocks that would other have been a hassle to avoid.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by IxGamerXL** [comment]
This is actually a pretty good idea. So long as the range consists slightly less than a Phase Bridge, it could probably be balanced and unique enough to be added.

**Comment by Hexrotor** [comment]
Good idea

**Comment by RealYeet** [comment]
Alternate idea, allow the phase bridge to move sideways 1 block for every 5 blocks it travels.

**Comment by Violet-Scarelli** [comment]
Honestly? I like this idea a lot. The ability to scootch a conveyor line over a few tiles diagonally would be REALLY helpful in a lot of cases, mostly in super-densely-packed factories.

**Comment by oinkymoo** [comment]
i agree that something so small should maybe not take surge so either make it longer or cheaper but otherwise i support this

**Comment by IxGamerXL** [comment]
> i agree that something so small should maybe not take surge so either make it longer or cheaper but otherwise i support this

Longer would be balanced along with it costing 5 surge alloy (and a few other types) similar to Phase Bridges requiring 5 phase fabric.

Keep in mind that Surge Bridges are practically power nodes that deliver items, (supposedly) transfers items nearly as fast as Phase Bridges, and would likely be used for some insane schematics.

Also, since they don't obey natural bridge rules, they would have to output from all four sides or utilize the power of Trigonometry, calculate PI and find the opposite direction of where a bridge is pointing at so that it doesn't output everywhere.

**Comment by itcannotbe** [comment]
~_Death sounds_~

![Screenshot_20220427_084721](https://user-images.githubusercontent.com/45280919/165416204-34f1921a-0b5c-45c8-aaf5-404d39982c2b.jpg)

**Comment by numer99** [comment]
what about unit transportation?
can't we use that?

**Comment by Violet-Scarelli** [comment]
Maybe instead of omnidirectional, they're octodirectional, being able to go diagonally as well at the cost of some overall bridge range.
